scss/card-filters.scss

```diff
@@ -64,11 +114,14 @@
     }
   }
   &-description {
-    font-size: $font-down-1;
-    display: -webkit-box;
-    -webkit-line-clamp: 2;
-    -webkit-box-orient: vertical;  
-    overflow: hidden;
+    margin: 0;
+    p {
+      font-size: $font-down-1;
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
+      -webkit-box-orient: vertical;
```

Comment:
Also, that's some outdated code! 😅 We have flex available for some time now!

Reply:
I believe this is required for line-camp to work properly.

Reply:
Ah, I see. We're close to `line-clamp` being widely supported, but not quite there yet. https://caniuse.com/?search=line-clamp